README.md

```diff
@@ -195,8 +195,8 @@ Module Long Term Support (LTS)](http://github.com/CloudNativeJS/ModuleLTS) polic
 
 | Version | Status          | Published | EOL      |
 | ------- | --------------- | --------- | -------- |
-| 6.x     | Active LTS      | Apr 2018  | Dec 2019 |
-| 5.x     | Maintenance LTS | Sep 2017  | Dec 2019 |
+| 6.x     | Active LTS      | Apr 2018  | Dec 2020 |
+| 5.x     | Maintenance LTS | Sep 2017  | Dec 2020 |
```

Comment:
On the second thought, do we have to extend EOL for v5.x?

IMO, after Dec 2019, v5.x should go EOL and v6.x should enter Maintenance LTS.

Thoughts?

Reply:
ah. I wasn't giving too much about it but to move everything from "Dec 2019" to "Dec 2020".  I'm good with keeping 5.x unchanged (i.e. EOL Dec 2019) and having 6.x EOL changed to Dec 2020.